eslint-bridge/tests/tools/sonar-runtime/rule/no-missing-sonar-runtime.ts

```diff
@@ -116,7 +116,11 @@ function getModuleNameOfImportedIdentifier(
     } else {
       maybeRequireCall = writeExpression;
     }
-    return getModuleNameFromRequire(maybeRequireCall);
+    const fqn = getFullyQualifiedName(context, maybeRequireCall, []);
+    if (fqn === null) {
+      return null;
+    }
+    return fqn.split('.')[0];
```

Comment:
The function mixes `null` and `undefined` in its return value. It would be simpler to stick to `undefined` and this change would become:
```suggestion
    const fqn = getFullyQualifiedName(context, maybeRequireCall);
    return fqn?.split('.')[0];
```

Reply:
Indeed, thanks for the suggestion!